core-bundle/tests/Functional/DbafsTest.php

```diff
@@ -106,6 +106,18 @@ public function testAutomaticallySyncsFiles(): void
         $this->assertCount(0, $contents);
     }
 
+    public function testCreateAndAccessFile(): void
+    {
+        // Make the DBAFS cache, that 'foo' does not exist
```

Comment:
```suggestion
        // Make sure that "foo" does not exist in the DBAFS cache
```

Reply:
No, I actually meant, that the DBAFS should cache a `null` entry, meaning "this resource does not exist".